homeassistant/components/maxcube/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "maxcube",
   "name": "eQ-3 MAX!",
   "documentation": "https://www.home-assistant.io/integrations/maxcube",
-  "requirements": ["maxcube-api==0.1.0"],
+  "requirements": ["maxcube-api==0.3.0"],
   "codeowners": []
```

Comment:
If you're going to pick up development of the maxcube, do you want to set yourself as a code owner? You can do that in a new PR.